dashboard/test/ui/runner.rb

```diff
@@ -232,6 +232,7 @@ def format_duration(total_seconds)
   arguments += " -t ~@chrome" if browser['browserName'] != 'chrome' && !$options.local
   arguments += " -t ~@no_safari" if browser['browserName'] == 'Safari'
   arguments += " -t ~@no_firefox" if browser['browserName'] == 'firefox'
+  arguments += " -t ~@no_mobile_safari" if browser['browserName'] == 'iphone'
```

Comment:
I can change it up. @no_safari blocks 'Safari' and 'iphone', @no_mobile_safari just blocks 'iphone'. Browser name for both iphone and ipad is "iphone"